scripts/thief.bicep

```diff
@@ -0,0 +1,346 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for
+// full license information.
+
+param location string
```

Comment:
Adding atleast metadata for params is good. Defaults can be optional.

param location string {
  default: '${resourceGroup().location'
  metadata: {
    description: 'The location of the resource.'
  }
}